tests/utils/blockchain.py

```diff
@@ -70,53 +72,47 @@ class TesterBlockchain(BlockchainDeployerInterface):
     Blockchain subclass with additional test utility methods and options.
     """
 
-    __test__ = False # prohibit Pytest from picking it up
-
-    _instance = None
+    __test__ = False  # prohibit pytest from collecting this object as a test
 
-    GAS_STRATEGIES = {**BlockchainDeployerInterface.GAS_STRATEGIES,
-                      'free': free_gas_price_strategy}
+    # Solidity
+    SOURCES: Tuple[SourceBundle, ...] = (
+        *BlockchainDeployerInterface.SOURCES,
+        SourceBundle(base_path=TEST_SOLIDITY_SOURCE_ROOT)
+    )
```

Comment:
I think this should be one source bundle (same as BlockchainDeployerInterface.SOURCES), and I'm still for variable to hold source bundles :)